scanner.py

```diff
@@ -42,12 +49,21 @@ def light(color):
 
 if __name__ == '__main__':
     try:
-        while True:
-            uniqueid = input("Insert UID: ").upper().strip()
-            if check_uid(uniqueid):
-                light(GREEN)
-            else:
-                light(RED)
-
-    except KeyboardInterrupt:
-        print("\nExiting")
+        while reading:
+            print("\nScan Card")
+            # Fetching the card UID and converting it to HEX
+            card_id = f'{reader.read_id():x}'.upper()
```

Comment:
```suggestion
            card_id = f'{reader.read_id():X}'
```

ez trick to make it uppercase xd

Reply:
Perfect!!